specification/resources/reserved_ipv6/reservedIPv6s_create.yml

```
@@ -0,0 +1,47 @@
+operationId: reservedIPv6s_create
```

```suggestion
operationId: reservedIPv6_create
```

Unfortunately Autorest, the tool we use to generate the Python client, does not handle this well. It ends up with a signature like `client.reserved_ipv6_s.create()` I've spent some time trying to investigate ways to work around this and have not found a good solution.

I think our best bet here is to just go with `reservedIPv6_create` which becomes `client.reserved_ipv6.create()`. It's a bit inconsistent to not be plural, but I think it is a cleaner approach. 

Thoughts?  (This applies to all of the operation IDs in this PR.)

---

Yes, makes sense, will update